### PR TITLE
Clean-up skipped tests in Roslyn.Compilers.VisualBasic.Syntax.UnitTests.DLL

### PR DESCRIPTION
**Pull request by AlekseyTs**
There is a small product change to fix https://devdiv.visualstudio.com/defaultcollection/DevDiv/_workitems?_a=edit&id=111538 and enable corresponding unit-test.

@dotnet/roslyn-compiler Please review.
CC @srivatsn

### CODE REVIEW

**Comment by AlekseyTs** [comment]
@dotnet/roslyn-compiler Please review.

**Comment by AlekseyTs** [comment]
@dotnet/roslyn-compiler Please review.

**Comment by jcouv** [comment]
LGTM

**Comment by AlekseyTs** [comment]
@dotnet/roslyn-compiler Please review, I need a second sign-off due to a small product change.

File: src/Compilers/Test/Utilities/VisualBasic/ParserTestUtilities.vb [hunk 1]
```diff
@@ -813,7 +816,10 @@ Public Module VerificationHelpers
...
                     InternalVerifyNoAdjacentTriviaHaveSameKind(tr.GetStructure)
                 End If
-                If prev IsNot Nothing Then
+
+                ' Based on http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems?_a=edit&id=527553
+                ' it is Ok to have adjacent SkippedTokensTrivias
+                If tr.Kind <> SyntaxKind.SkippedTokensTrivia AndAlso prev IsNot Nothing Then
                     Assert.True(prev.Value.Kind <> tr.Kind,
                                 "Both current and previous trivia have Kind=" & tr.Kind.ToString &
                                 " [See under TokenKind=" & node.Kind().ToString & ", NonTerminalKind=" & node.Parent.Kind.ToString & "]")
```
Comment (cston) [inline src/Compilers/Test/Utilities/VisualBasic/ParserTestUtilities.vb]: Are the two loops identical? If so, consider extracting a method.

File: src/Compilers/VisualBasic/Portable/Preprocessor/ExpressionEvaluator.vb [hunk 2]
```diff
@@ -852,15 +852,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
...
 
             If specialType = SpecialType.System_Object AndAlso Not IsNothing(val) Then
-                Return ReportSemanticError(ERRID.ERR_CannotConvertValue2, expr)
+                Return ReportSemanticError(ERRID.ERR_UnaryOperand2, expr, expr.OperatorToken.ValueText, specialType.GetDisplayName())
             End If
 
             If specialType = SpecialType.System_Char OrElse specialType = SpecialType.System_DateTime Then
-                Return ReportSemanticError(ERRID.ERR_CannotConvertValue2, expr)
+                Return ReportSemanticError(ERRID.ERR_UnaryOperand2, expr, expr.OperatorToken.ValueText, specialType.GetDisplayName())
             End If
```
Comment (cston) [inline src/Compilers/VisualBasic/Portable/Preprocessor/ExpressionEvaluator.vb]: Consider combining all three cases with ```If A OrElse B OrElse C Then```. (C already has two cases.) Or use Select Case for all cases including `SpecialType.None`.

**Comment by AlekseyTs** [comment]
Test prtest/win/dbg/unit32 please.
http://dotnet-ci.cloudapp.net/job/roslyn_prtest_win_dbg_unit32/5455/

**Comment by AlekseyTs** [comment]
@cston I addressed your feedback.

File: src/Compilers/Test/Utilities/VisualBasic/ParserTestUtilities.vb [hunk 3]
```diff
@@ -798,31 +798,31 @@ Public Module VerificationHelpers
...
-                    InternalVerifyNoAdjacentTriviaHaveSameKind(tr.GetStructure)
-                End If
-                If prev IsNot Nothing Then
-                    Assert.True(prev.Value.Kind <> tr.Kind,
-                                "Both current and previous trivia have Kind=" & tr.Kind.ToString &
-                                " [See under TokenKind=" & node.Kind().ToString & ", NonTerminalKind=" & node.Parent.Kind.ToString & "]")
-                End If
+                VerifyNoAdjacentTriviaHaveSameKind(node, tr, prev)
                 prev = tr
             Next
```
Comment (cston) [inline src/Compilers/Test/Utilities/VisualBasic/ParserTestUtilities.vb]: Can we move the entire `For Each` into `VerifyNoAdjacentTrivia`?

Comment (AlekseyTs) [inline src/Compilers/Test/Utilities/VisualBasic/ParserTestUtilities.vb]: I think there is a bug in the second loop, it should probably go over TrailingTrivia. I'll fix that.

**Comment by cston** [comment]
LGTM

**Comment by AlekseyTs** [comment]
Test this please

**Pull request merged**